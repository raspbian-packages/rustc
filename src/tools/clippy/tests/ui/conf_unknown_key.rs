// error-pattern: error reading Clippy's configuration file: unknown key `foobar`

#![feature(plugin)]
#![plugin(clippy(conf_file="./tests/auxiliary/conf_unknown_key.toml"))]

fn main() {}
