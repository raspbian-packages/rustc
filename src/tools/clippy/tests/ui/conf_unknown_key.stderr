error[E0658]: compiler plugins are experimental and possibly buggy (see issue #29597)
 --> $DIR/conf_unknown_key.rs:4:1
  |
4 | #![plugin(clippy(conf_file="../auxiliary/conf_unknown_key.toml"))]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: add #![feature(plugin)] to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
